packages/engine/src/scene/components/MediaComponent.ts

```diff
@@ -331,6 +331,12 @@ export function MediaReactor() {
 
   useEffect(
     function updateMediaElement() {
+      if (!media.ended.value) {
+        // If current track is not ended, don't change the track
+        return
```

Comment:
how would one force a certain track to play?

Reply:
We can't do that. We can't do that currently as well.

Reply:
Shall I remove it?